reconcile/utils/ocm/labels.py

```diff
@@ -30,6 +33,42 @@ def get_subscription_labels(
             yield subscription_label
 
 
+def add_subscription_labels(
+    ocm_api: OCMBaseClient,
+    cluster: ClusterDetails,
+    labels: Mapping[str, str],
+) -> None:
+    """Add the given labels to the cluster subscription."""
+    for key, value in labels.items():
+        ocm_api.post(
+            api_path=f"{cluster.ocm_cluster.subscription.href}/labels",
+            data={"kind": "Label", "key": key, "value": value},
+        )
+
+
+def update_subscription_labels(
+    ocm_api: OCMBaseClient,
+    cluster: ClusterDetails,
+    labels: Mapping[str, str],
```

Comment:
same here, `OCMCluster` as the cluster object and the `OCMSubscriptionLabel` objects

Reply:
`OCMCluster` has just a `subscription: OCMModelLink`

Reply:
i think you don't need the cluster at all if you pass in `OCMSubscriptionLabel` objects since they have the `href` for the update.

Reply:
I refactored all three label methods. Please take a look at [f012647](https://github.com/app-sre/qontract-reconcile/pull/3714/commits/f01264740e1ab2a07496fa3e526697821ccae267)